src/components/Dropdown/Dropdown.tsx

```diff
@@ -265,6 +269,8 @@ export interface DropdownProps {
   placeholderRef?: React.RefObject<HTMLElement>;
   closeIconRef?: React.RefObject<HTMLElement>;
   arrowIconRef?: React.RefObject<HTMLElement>;
+  searchContainerRef?: React.RefObject<HTMLDivElement>;
+  inputRef?: React.RefObject<HTMLElement>;
```

Comment:
searchInputRef?

Reply:
agreed ^^

Reply:
It seems like this isn't being used anyway?

Reply:
Had to modify the TextInput type for its refs. I left it as inputRef. The internal one was searchInputRef. I figured best to leave that as is. Thoughts?